engine/src/configuration/configuration.h

```diff
@@ -0,0 +1,95 @@
+#ifndef CONFIGURATION_H
+#define CONFIGURATION_H
+
+
+struct Configuration
+{
+    // Config Structs Declaration
+
+    // These are settings that are relevant to collision.cpp and are marked as hacks
+    // or their comments are unclear or they are unclear
+    struct CollisionHacks {
+        const float collision_hack_distance;
+        const bool collision_damage_to_ai;
+        const bool crash_dock_hangar;
+        const bool crash_dock_unit;
+        const float front_collision_hack_angle;
+        const float front_collision_hack_distance;
+        const bool cargo_deals_collide_damage;
+        //const int upgrade_faction = (cargo_deals_collide_damage ? -1 : FactionUtil::GetUpgradeFaction())
+
+        CollisionHacks();
+    };
+
+    // Also radar
+    struct Computer {
+        // Note: below is probably a stale comment
+        // DO NOT CHANGE see unit_customize.cpp
+        const float default_lock_cone;
+        const float default_max_range;
+        const float default_tracking_cone;
+
+        Computer();
+    };
+
+    struct Fuel {
+        const float afterburner_fuel_usage;
+        const float fmec_exit_velocity_inverse; // Well, that's very clear
```

Comment:
Re: Comment block - there are a pair of "FMEC" variables - they both involve "Fuel Mass to Energy Conversion" - this one happens to specify the inverse (it's only ever used as 1/Value, so just encode 1/Value, not Value) of the assumed exit velocity of the mass ejected as thrust, calculated based on energy-possible (if not necessarily plausible) outcomes from a Li-6 + Deuterium fusion reaction.

The other variable (not present here) FMEC_factor, is used in reactor --> energy production. As the comment in the code next to the variable init says, it specifies how many metric tons (1 VS mass unit) of fuel are used to produce 100MJ (one units.csv energy recharge unit) of recharge.

At some point, it wouldn't kill us to renormalize the engine and dataset to both just use SI units, but that's not a priority.

Reply:
I will add this info. Thanks.